### PR TITLE
[Generator] Update documentation

### LINKED ISSUE #3468: Customizing VA using TypeScript
/virtual-assistant/tutorials/customize-assistant/typescript/2-edit-your-greeting/

I installed the bot virtual assistant template using the yo bot-virtualassistant command but could not find either src/content or src/locales directories. Are these directory structures generated by yo's VA template? I tried to run yo's VA generator but couldn't find src/content or src/locate directories in it. Looks to be a bug with either the template or the documentation

### PR DESCRIPTION
**Pull request by Batta32**
Fix #3468

### Purpose
*What is the context of this pull request? Why is it being done?*
The generator's documentation was outdated.

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*
Improve the generator's documents following the C# ones.

### Tests
*Is this covered by existing tests or new ones? If no, why not?*
\-

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*
\-

#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [ ] I have added or updated the appropriate tests	
- [X] I have updated related documentation

### CODE REVIEW

File: docs/_docs/skills/tutorials/create-skill/typescript/3-create-your-skill.md [hunk 1]
```diff
@@ -50,18 +44,18 @@ yo bot-virtualassistant:skill
...
 |-----------------------------------|--------------------------------------------------------------------------------------------------------------|
-| -n, --skillName <name>              | name of new skill (by default takes `customSkill`)                                                          |
-| -d, --skillDesc <description>       | description of the new skill (by default is empty) |
-| -l, --skillLang <array of languages>| languages for the new skill. Possible values are `de`, `en`, `es`, `fr`, `it`, `zh` (by default takes all the languages)|
-| -p, --skillGenerationPath <path>    | destination path for the new skill (by default takes the path where you are running the generator)            |
-| --noPrompt                        | indicates to avoid the prompts                                                                               |
+| -n, --skillName <name>              | The name you want to give to your skill (by default takes `sample-skill`)                                                          |
+| -d, --skillDesc <description>       | A brief bif of text used to describe what your skill does (by default is empty) |
```
Comment (pavolum) [inline docs/_docs/skills/tutorials/create-skill/typescript/3-create-your-skill.md]: bit*

File: docs/_docs/skills/tutorials/create-skill/typescript/6-run-your-skill.md [hunk 2]
```diff
@@ -4,17 +4,17 @@ category: Skills
...
-order: 5
+order: 6
 ---
 
 # Tutorial: {{page.subcategory}} ({{page.language}})
 
 ## {{ page.title }}
 
-Once deployment is complete, you can start debugging through the following steps:
+After deployment, you and run and test your Skill project using these steps:
```
Comment (pavolum) [inline docs/_docs/skills/tutorials/create-skill/typescript/6-run-your-skill.md]: can*

File: docs/_docs/virtual-assistant/tutorials/create-assistant/typescript/5-run-your-assistant.md [hunk 3]
```diff
@@ -10,7 +10,8 @@ order: 5
 # Tutorial: {{page.subcategory}} ({{page.language}})
 
 ## Run your assistant
-When deployment is complete, you can run your Virtual Assistant through the following steps:
+
+After deployment, you and run and test your Virtual Assistant project using these steps:
```
Comment (pavolum) [inline docs/_docs/virtual-assistant/tutorials/create-assistant/typescript/5-run-your-assistant.md]: can*

**Review by pavolum — APPROVED** [review]

**Comment by Batta32** [comment]
Thanks @pavolum for the feedback. I've updated the documents following your recomendations 😊.

File: docs/_docs/skills/tutorials/create-skill/typescript/2-download-and-install.md [hunk 4]
```diff
@@ -11,28 +11,30 @@ order: 2
...
 ## {{ page.title }}
 
-> It's important to ensure all of the following prerequisites are installed on your machine prior to attempting deployment otherwise you may run into deployment issues.
+## Download and install
 
+1. Install the bot-virtualassistant generator
+
+   ```bash
+      npm install -g generator-bot-virtualassistant
+   ```
```
Comment (esoler-sage) [inline docs/_docs/skills/tutorials/create-skill/typescript/2-download-and-install.md]: This can only be done after installing npm, isn't it?

Comment (Batta32) [reply docs/_docs/skills/tutorials/create-skill/typescript/2-download-and-install.md]: Yes @esoler-sage, I fixed those documents. Thanks for the feedback 😊.

File: docs/_docs/virtual-assistant/tutorials/create-assistant/typescript/4-provision-you-azure-resources.md [hunk 5]
```diff
@@ -50,4 +50,4 @@ To deploy your Assistant using the Azure Resource Manager (ARM) template provide
 
 You can find more detailed deployment steps including customization instructions in the [Deployment Scripts reference]({{site.baseurl}}/virtual-assistant/handbook/deployment-scripts/). 
 
-> For manual deployment steps, refer to the [Deploy using CLI tools]({{site.baseurl}}/virtual-assistant/tutorials/deploy-assistant/cli/1-intro/) and [Deploy using web]({{site.baseurl}}/virtual-assistant/tutorials/deploy-assistant/web/1-intro/) tutorials.
\ No newline at end of file
+> For manual deployment steps, refer to the [Deploy using CLI tools]({{site.baseurl}}/virtual-assistant/tutorials/deploy-assistant/cli/1-intro/) and [Deploy using web]({{site.baseurl}}/virtual-assistant/tutorials/deploy-assistant/web/1-intro/) tutorials.
```
Comment (taicchoumsft) [inline docs/_docs/virtual-assistant/tutorials/create-assistant/typescript/4-provision-you-azure-resources.md:53]: Minor nit: doesn't look like any changes to this file? Remove from checkin?

**Review by taicchoumsft — APPROVED** [review]
:shipit:

**Pull request merged**